npm/src/index.js

```diff
@@ -21,9 +23,66 @@ export const bin = {
     : path.join(env.PREFIX, 'bin', 'ssc')
 }
 
+export const firstTimeExperienceSetup = async () => {
```

Comment:
I feel like we can improve naming here to make it more clear what the return value mean

Reply:
I'm open to suggestions. It was initially just an action method, now it's returning the status to indicate whether or not setup is complete enough to run `ssc`: On windows the min requirement is for `VC_Runtime140.dll` to be installed.

`firstTimeExperienceSetupIsValid`?

Reply:
maybe something like `areSetupPrerequisitesMet`?

Reply:
I'm not keen on something like `areSetupPrerequisitesMet` because it doesn't inform that FTE is going to be triggered as part of this function.

Reply:
`firstTimeExperienceSetup` sounds find to me